src/Symfony/Component/Form/ChoiceList/Factory/ChoiceListFactoryInterface.php

```diff
@@ -80,6 +80,7 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, ?callable $v
      * @param bool                $duplicatePreferredChoices  Whether the preferred choices should be duplicated
      *                                                        on top of the list and in their original position
      *                                                        or only in the top of the list
+     * @param array|callable|null $help                       The callable generating the helps
      */
-    public function createView(ChoiceListInterface $list, array|callable|null $preferredChoices = null, callable|false|null $label = null, ?callable $index = null, ?callable $groupBy = null, array|callable|null $attr = null, array|callable $labelTranslationParameters = [], bool $duplicatePreferredChoices = true): ChoiceListView;
+    public function createView(ChoiceListInterface $list, array|callable|null $preferredChoices = null, callable|false|null $label = null, ?callable $index = null, ?callable $groupBy = null, array|callable|null $attr = null, array|callable $labelTranslationParameters = [], bool $duplicatePreferredChoices = true, array|callable|null $help = null): ChoiceListView;
```

Comment:
Adding a new parameter in an interface is a BC break. You need to add only the `@param` and add it in implementations, following our process to add a new argument progressively (the first step will trigger a deprecation warning if you make an implementation that does not have the new argument)